src/utils/generatePods.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { map } from 'lodash'
+
+export const generatePods = (uid, pods) => {
+  let podsData = {}
+  let podsPending = []
+  let podsList = []
+  let podsTooltip = []
+
+  if (pods && pods[uid]) {
```

Comment:
You can use this instead of `&&`:
```suggestion
  if (pods?.[uid]) {
```